docs/integrations-guides/custom_security.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Custom Security
+sidebar_position: 5
+---
+
+# Custom Quorums And Thresholds
+
+EigenDA allows users to tailor their data availability solution while maintaining security guarantees.
+
+This is done by lettings users define their own [custom quorum](../core-concepts/security/security-model.md#quorums-and-security-models) and [security thresholds](../core-concepts/security/security-model.md#safety-and-liveness-analysis). 
+Rollups that do so must enforce that the DA Certificate they receive from the disperser meets the `thresholds` they have set for each quorum, including their custom quorum.
+
+Dispersing to a custom quorum effectively additionally replicates the data to the set of operators which hold the custom token that defines the custom quorum.
```

Comment:
Is a custom quorum token always ERC20?

Reply:
tbh I don't really know. In theory no (see new operatorSet terminology), but in practice I'm pretty sure it is, and that's what we are selling. I think its best to keep it simple, but this is above my paygrade.